packages/@aws-cdk/aws-ecs/lib/base/task-definition.ts

```diff
@@ -713,7 +730,7 @@ export interface DockerVolumeConfiguration {
    *
    * @default No labels
    */
-  readonly labels?: string[];
+  readonly labels?: { [key: string]: string; };
```

Comment:
This type hasn't changed over the CFN Spec update. It was incorrect already, but I've uncovered the bug because it is no longer passed through an `any`-typed cross-point.